website/pages/use-cases/automated-service-networking-with-consul.jsx

```diff
@@ -71,8 +71,10 @@ export default function AutomatedServiceNetworkingWithConsulPage() {
         }}
       />
 
-      <FeaturedSliderSection
+      <FeaturedSlider
         heading="Case Studies"
+        theme="dark"
+        product="nomad"
```

Comment:
The local component effectively had these hardcoded, needed to add them to get the global component to render identically.